components/identities/actions/transferFormModal.tsx

```diff
@@ -32,34 +35,31 @@ const TransferFormModal: FunctionComponent<TransferFormModalProps> = ({
   const router = useRouter();
   const { address } = useAccount();
   const { tokenId } = router.query;
-  const numId = parseInt(tokenId as string);
   const { addTransaction } = useNotificationManager();
   const [addressInput, setAddressInput] = useState<string>("");
   const { starknetIdNavigator } = useContext(StarknetIdJsContext);
   const [isTxSent, setIsTxSent] = useState(false);
 
-  //set_domain_to_address execute
-  const transfer_identity_and_set_domain_multicall = [
-    {
-      contractAddress: process.env.NEXT_PUBLIC_NAMING_CONTRACT as string,
-      entrypoint: "set_domain_to_address",
-      calldata: [...callDataEncodedDomain, hexToDecimal(targetAddress ?? "")],
-    },
-    {
-      contractAddress: process.env.NEXT_PUBLIC_STARKNETID_CONTRACT as string,
-      entrypoint: "transferFrom",
-      calldata: [
-        hexToDecimal(address ?? ""),
-        hexToDecimal(targetAddress ?? ""),
-        numId,
-        0,
-      ],
-    },
-  ];
+  // todo: clear target address if necessary
+  // {
+  //   contractAddress: process.env.NEXT_PUBLIC_NAMING_CONTRACT as string,
+  //   entrypoint: "set_domain_to_address",
+  //   calldata: [...callDataEncodedDomain, hexToDecimal(targetAddress ?? "")],
+  // },
 
+  const transfer_id_call = {
+    contractAddress: process.env.NEXT_PUBLIC_STARKNETID_CONTRACT as string,
+    entrypoint: "transferFrom",
+    calldata: [
+      hexToDecimal(address ?? ""),
+      hexToDecimal(targetAddress ?? ""),
+      identity?.id as string,
+      0,
+    ],
+  };
```

Comment:
in the `identityChangesCalls.tsx` to me as it concerns the identity

Reply:
Done, see: https://github.com/starknet-id/app.starknet.id/pull/580/commits/ab004abb1ed5639dd7a7fc21cb7b8a6aab1aa603